### DISCUSSION TITLE
The ability to search for answers on a specific website or websites of an organization

### DISCUSSION THREAD

**Discussion by GeorgeStevens13** [microsoft/PubSec-Info-Assistant#824] [Q&A]
Is the ability to limit the web search component of the accelerator to only search on specific websites. For instance customer A wants to find information only on https://www.customerA.site and/or https://www.customerB.site. RAG is not really an option since the current sites have too many levels down and the combined total size is too big for ingestion.

I have attempted to use the site:[website] option for the Bing api component, and I have changed the system prompt and having some success but was wondering if anybody else had a better way?

**Comment by bjakems** [comment]
Hi @GeorgeStevens13 ,
Yes, I recommend your approach for limiting sites to a select few.  
For Bing API references on the syntax to limit sites to multiple subsites, please reference the site: parameter at https://learn.microsoft.com/en-us/bing/search-apis/bing-web-search/filter-answers and https://support.microsoft.com/en-us/topic/advanced-search-keywords-ea595928-5d63-4a0b-9c6b-0b769865e78a.

**Reply by GeorgeStevens13** [reply]
I have figured this out thank you.
